Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -390,6 +390,14 @@ lemma integral_toReal_rnDeriv [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪
     ∫ x, (μ.rnDeriv ν x).toReal ∂ν = (μ Set.univ).toReal := by
   rw [← integral_univ, setIntegral_toReal_rnDeriv hμν Set.univ]
 
+lemma integral_toReal_rnDeriv' {α : Type*} {m : MeasurableSpace α}
+    {μ : Measure α} {ν : Measure α} [IsFiniteMeasure μ] [SigmaFinite ν] :
```

Comment:
Certainly `α` and the measures are already defined in a `variable` line somewhere above?
```suggestion
lemma integral_toReal_rnDeriv' [IsFiniteMeasure μ] [SigmaFinite ν] :
```